project/Build.scala

```diff
@@ -63,8 +63,6 @@ object MyScalaJSPlugin extends AutoPlugin {
   val isGeneratingEclipse =
     Properties.envOrElse("GENERATING_ECLIPSE", "false").toBoolean
 
-  val wantSourceMaps = settingKey[Boolean]("Whether source maps should be used")
```

Comment:
Er, we really cannot do that. [It's used in the community build.](https://github.com/scala/community-builds/blob/c1e7bb720d876954d94207939b3367ae2c6f9f5c/configs/community.dbuild#L427). :-s